litex/tools/litex_sim.py

```diff
@@ -62,6 +62,72 @@ class Platform(SimPlatform):
     def __init__(self):
         SimPlatform.__init__(self, "SIM", _io)
 
+# DFI PHY model settings ---------------------------------------------------------------------------
+
+sdr_phy_settings = {
+    "nphases": 1,
+    "rdphase": 0,
+    "wrphase": 0,
+    "rdcmdphase": 0,
+    "wrcmdphase": 0,
+    "cl": 2,
+    "read_latency": 4,
+    "write_latency": 0
+}
+
```

Comment:
This should also be integrated in get_sdram_phy_settings.

Reply:
This is now part of `get_sdram_phy_settings`